public/locations-list-map.js

```diff
@@ -195,6 +201,17 @@ function ctn(text) {
   return document.createTextNode(text);
 }
 
+// Turns a string with embedded \n characters into an Array of text nodes separated by <br>
+function multilineStringToNodes(input) {
+  const textNodes = input.split('\n').map((s) => ctn(s));
```

Comment:
Stupid jscript trick. You can eliminate the foreach down below and turn this into a one-liner:
```
return [].concat(...input.split('\n').map((s) => [ ctn(s), ce('br') ]));